editor/src/document/properties_panel_message_handler.rs

```diff
@@ -154,6 +156,13 @@ impl<'a> MessageHandler<PropertiesPanelMessage, PropertiesPanelMessageHandlerDat
 				);
 				self.active_selection = None;
 			}
+			Init => responses.push_back(
+				BroadcastMessage::SubscribeSignal {
+					on: BroadcastSignal::SelectionChanged,
+					send: Box::new(PropertiesPanelMessage::UpdateSelectedDocumentProperties.into()),
```

Comment:
I think the into() should take care of the boxing so you don't need to write that out explicitly.
But I'm not sure if that would require type hints so feel free to ignore